api/src/main/resources/workbench-api.yaml

```diff
@@ -5345,6 +5384,14 @@ definitions:
         format: int64
         description: 'Timestamp when the user first changed from Unregistered to Registered
           data access level. May be null if the user has never done so.'
+  AccountLinkInfo:
+    description: 'User account linking information. For now, we just need eRA Commons username,
```

Comment:
does this validate?  You may need a line continuation character here

Reply:
Added some. PTAL

Reply:
What I meant was: will Swagger allow this comment to span multiple lines?

But I tested this and it is fine.  Never mind.